src/lib.rs

```diff
@@ -161,7 +160,7 @@ pub struct Queue<'a> {
 pub enum BindingResource<'a> {
     Buffer {
         buffer: &'a Buffer,
-        range: Range<u32>,
+        range: Range<u64>,
```

Comment:
we should use the [BufferAddress](https://github.com/gfx-rs/wgpu/blob/a056d2478a3e34fa20071957af89fc6eda7a313d/wgpu-native/src/lib.rs#L53) alias for these

Reply:
Are there other instances besides this one that should use the BufferAddress alias?

Reply:
Yeah okay, it's a lot more, it would have been obvious had I actually looked at wgpu more closely!

Reply:
Actually just %s/u64/BufferAddress/g was correct, I made it harder than it had to be :P